### ISSUE TITLE
After initialising a values repo, I can't start Otomi Console

### ISSUE THREAD

**Issue by svatwork** [redkubes/otomi-core#208]
**Describe the bug:**
In otomi-values/bin/otomi the script refers to otomi-values/bin/console.sh as if it's in otomi-values after executing the one-liner found on [this page](https://redkubes.github.io/otomi/docs/setup). It doesn't exist, and therefore throws an error. 

**To Reproduce**
1. 

```
# initialize git repo
mkdir otomi-values && cd $_ && git init .
# and get all the files
docker run -e ENV_DIR=$PWD -v $PWD:$PWD otomi/core:latest bin/bootstrap.sh
# and source the aliases including the otomi cli
. bin/aliases
```

2.

```
$ otomi console
bin/otomi: line 270: bin/console.sh: No such file or directory
```

**Expected behavior:**
```
$ otomi console
OTOMI_PULLSECRET not given!
```

**Comment by Morriz** [comment]
Did you follow the readme and obtain a pullsecret and then run the bootstrap again?

**Comment by Morriz** [comment]
solved with the release of v0.11.40

**Comment by Morriz** [comment]
OTOMI_PULLSECRET is not needed anymore in env, it is only expected in 'secrets.settings.yaml`
